### PR TITLE
feat: start instance if necessary

### PR DESCRIPTION
**Pull request by daknhh**
-check if ec2 instance is running
- start intance
- adjust console log

### CODE REVIEW

**Comment by qoomon** [comment]
@daknhh I like the idea, however it would be nice if you can move your code into a file `aws-ssm-ec2-proxy-command-start-instance.sh` next to `aws-ssm-ec2-proxy-command.sh` and then within `aws-ssm-ec2-proxy-command-start-instance.sh` call `exec aws-ssm-ec2-proxy-command.sh` after target instance has started.

File: aws-ssm-ec2-iam-policy.json
Comment (qoomon) [inline aws-ssm-ec2-iam-policy.json:1]: Please move additional permissions to an separate statement(s) to make it easier to understand what belongs to that extra feature and what to the base ssm functionality. In addition please set SIds for all Statements to give a hint for the scope of the statement

Comment (daknhh) [reply aws-ssm-ec2-iam-policy.json:1]: added new iam policy file with Sids

File: aws-ssm-ec2-proxy-command.sh
Comment (qoomon) [inline aws-ssm-ec2-proxy-command.sh:1]: I think this file should kept untouched

File: aws-ssm-ec2-proxy-command-start.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env sh
+######## Source ################################################################
+#
+# https://github.com/qoomon/aws-ssm-ec2-proxy-command
+#
+######## Usage #################################################################
+# https://github.com/qoomon/aws-ssm-ec2-proxy-command/blob/master/README.md
+#
+# Install Proxy Command
+#   - Move this script to ~/.ssh/aws-ssm-ec2-proxy-command.sh
```
Comment (qoomon) [inline aws-ssm-ec2-proxy-command-start.sh]: I think this is misleading because this is not referring to this very own script itself

Comment (qoomon) [reply aws-ssm-ec2-proxy-command-start.sh]: Just describe what is need in addition to ~/.ssh/aws-ssm-ec2-proxy-command.sh
For usage steps of ~/.ssh/aws-ssm-ec2-proxy-command.sh refer to the usage steps of ~/.ssh/aws-ssm-ec2-proxy-command.sh

Comment (daknhh) [reply aws-ssm-ec2-proxy-command-start.sh]: adjusted :)

File: aws-ssm-ec2-proxy-command.sh
Comment (daknhh) [reply aws-ssm-ec2-proxy-command.sh:1]: revert to original file ✅

File: aws-ssm-ec2-proxy-command-start-instance.sh [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+#
+# https://github.com/qoomon/aws-ssm-ec2-proxy-command
+#
+######## Usage #################################################################
+# https://github.com/qoomon/aws-ssm-ec2-proxy-command/blob/master/README.md
+#
+# Install Proxy Command
+#   - Check Install Steps for aws-ssm-ec2-proxy-command.sh
+#   - Move this script to ~/.ssh/aws-ssm-ec2-proxy-command-start-instance.sh
+#   - Ensure it is executable (chmod +x ~/.ssh/aws-ssm-ec2-proxy-command.sh)
```
Comment (qoomon) [inline aws-ssm-ec2-proxy-command-start-instance.sh:12]: should be ~/.ssh/aws-ssm-ec2-proxy-command-start-instance.sh I guess

**Comment by qoomon** [comment]
@daknhh  thanks a lot

**Comment by daknhh** [comment]
always

**Pull request merged**